homeassistant/components/voip/config_flow.py

```diff
@@ -3,10 +3,22 @@
 
 from typing import Any
 
+from voip_utils import SIP_PORT
+import voluptuous as vol
+
 from homeassistant import config_entries
 from homeassistant.data_entry_flow import FlowResult
 
-from .const import DOMAIN
+from .const import CONF_SIP_PORT, DOMAIN
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(
+            CONF_SIP_PORT,
+            description={"suggested_value": SIP_PORT},
```

Comment:
I think we can just use a default in this case?

```suggestion
            default=5060,
```